### DISCUSSION TITLE
Configuring DNS

### DISCUSSION THREAD

**Discussion by yllekz** [wg-easy/wg-easy#104] [Q&A] [answered]
Hey there, I was wondering if there was an easy way to get local LAN DNS working when running this container. Currently, I have to mod hosts files to map IPs to names when using Wireguard (which you can't really do on mobile devices or tablets) and I wasn't sure if there was a server-side way to configure this?

**Comment by domigi** [comment]
did you find any solution to this?

**Comment by pheiduck** [comment, marked as answer]
@yllekz yes, this is still possible add your Local DNS to `- WG_DEFAULT_DNS=<your dns>`
